johnmcl001/domus-api
# DOM-1: docker-compose.yml updated to use keycloak connect to postgres db

Basic setup of docker-compose for development
Sets up sonar qube, postgres and keycloak.
Testing:

1. From root execute docker compose up
2. Go to localhost:8080/auth, credentials are admin, admin
3. Create a user via the users tab on the left of the screen.
4. Open database localhost:5432/keycloak with credentials username: keycloak, password: password.
5. Verify user added in user entity table `select * from user_entity`
6. Open sonar qube at localhost:9000
7. In a separate terminal execute `./gradlew sonarqube`
8. Check that project shows up in sonar.
9. Approve PR.
